mappings/net/minecraft/world/decorator/impl/DecoratorDeposit.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS none/awc net/minecraft/world/decorator/impl/DecoratorDeposit
```

Comment:
This is named DecoratorDeposit and not DecoratorOre, reason being it's used for things like gravel and dirt as well as ores.